# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/xmms-infinity/settitle.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur infinity-0.2.orig/display.c infinity-0.2/display.c
--- infinity-0.2.orig/display.c	2000-07-01 01:53:07.000000000 +0200
+++ infinity-0.2/display.c	2003-09-29 00:41:29.000000000 +0200
@@ -415,6 +415,7 @@
 			fprintf(stderr, "Couldn't init video mode: %s\n", SDL_GetError());
 			exit(1);
 		}
+	SDL_WM_SetCaption("Infinity", NULL);
 	SDL_ShowCursor(0);
 	SDL_EnableKeyRepeat(0,0);
 
